packages/asset/contracts/AssetCreate.sol

```diff
@@ -424,6 +430,13 @@ contract AssetCreate is
 
         if (catalystBalance < mintData.amount) {
             require(matchedOrders.length > 0, "AssetCreate: No order data");
+            // DELEGATE CALL
+            // bytes4 selector = bytes4(keccak256("matchOrdersFrom(address,ExchangeMatch[])"));
+            // bytes memory data = abi.encodeWithSelector(selector, mintData.caller, matchedOrders);
+            // (bool success, ) = address(exchangeContract).delegatecall(data);
+            // require(success, "AssetCreate: Match orders failed");
+
+            // REGULAR
             exchangeContract.matchOrdersFrom(mintData.caller, matchedOrders);
```

Comment:
This section does not work and returns:

`Error: Transaction reverted: function selector was not recognized and there's no fallback function`

Both regular contract call through the interface nor `delegateCall` work.
The Exchange contract does have the function available on it. 

I'm totally flabbergasted.